pm/sendermonitor.go

```diff
@@ -234,11 +223,13 @@ func (sm *senderMonitor) cache(addr ethcommon.Address) {
 // startTicketQueueConsumerLoop initiates a loop that runs a consumer
 // that receives redeemable tickets from a ticketQueue and feeds them into
 // a single output channel in a fan-in manner
-func (sm *senderMonitor) startTicketQueueConsumerLoop(queue *ticketQueue, done chan struct{}) {
+func (sm *LocalSenderMonitor) startTicketQueueConsumerLoop(queue *ticketQueue, done chan struct{}) {
 	for {
 		select {
 		case ticket := <-queue.Redeemable():
-			sm.redeemable <- ticket
+			if err := sm.redeemWinningTicket(ticket); err != nil {
```

Comment:
Leaving this note for myself that we need to address [this comment](https://github.com/livepeer/go-livepeer/pull/1493#discussion_r434937684).

Reply:
Since we'd need to rebase and force push before we can address this please let me know when you have reviewed the remaining open change request so that I don't overwrite the existing review process by a force push.